pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,42 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+
+  src = fetchurl {
+    url = "https://download.ankama.com/launcher/full/linux";
+    hash = "sha256-aTzTQRwIpP87rn/GJ958y9nKbkuZ1wai2a4oNBLJ7bU=";
+    curlOptsList = [
+      "-A"
+      "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36"
+      "-L"
+    ];
+  };
+in
+
+appimageTools.wrapType2 {
+  inherit pname version src;
+  extraPkgs = pkgs: [ pkgs.wine ];
+
+  meta = {
+    description = "Ankama Launcher (read the long description for troubleshooting)";
+    longDescription = ''
+      Ankama Launcher is a portal that lets you access all of Ankama's many universes with just a single click! Connect to our video games, VOD animations, webtoons, and even our livestreams! Download updates, stay up to date with the latest news, and chat with your friends!
+
+      If you have a wine error when running Dofus, delete or rename the cinematics directory:
+      go to the directory where you installed the game and do
+      mv content/gfx/cinematics content/gfx/cinematics_DISABLE
+
+      Also don't use a vpn or proxy or you won't be able to download the source or connect to Ankama's servers.
+    '';
```

```suggestion
    description = "Ankama Launcher";
    longDescription = ''
      Ankama Launcher is a portal that allows you to access Ankama's video games, VOD animations, webtoons, and livestreams, as well as download updates, stay up to date with the latest news, and chat with your friends.

      If you encounter a `wine` error while running *Dofus*, delete or rename the `cinematics/` directory:
      - Go to the directory where you installed the game and run: `mv content/gfx/cinematics content/gfx/cinematics_DISABLE`
    '';
```
Some writing changes. The VPN information isn't necessary; VPN users are expected to know that when dealing with third-party servers.